packages/contracts/contracts/engine/V3/forks/GenArt721CoreV3_Engine_Flex_PROOF.sol

```diff
@@ -99,7 +99,7 @@ import "../../../libs/v0.8.x/Bytes32Strings.sol";
  * Additional admin and artist privileged roles may be described on minters,
  * registries, and other contracts that may interact with this core contract.
  */
-contract GenArt721CoreV3_Engine_Flex_PROOF is
+abstract contract GenArt721CoreV3_Engine_Flex_PROOF is
```

Comment:
I think I made these forks a different interface, so they don't need to be abstract anymore?